scripts/travis/build/build.sh

```diff
@@ -7,7 +7,7 @@ cd $DIR/../../../
 rm -rf tmp && mkdir tmp;
 
 npm install @alfresco/adf-cli@alpha
```

Comment:
do we still need this install?

Reply:
With `npx` I think it's not needed to install separately

Reply:
yep we need I don't know why but when I ask the npx to install the latest alpha it runs the audit script by default instead of calling the command I am passing. 
Probably we can remove it once we move the audit from js to ts

Reply:
have you try with  npx @alfresco/adf-cli@alpha ?

Reply:
Yep I have checked that command and as result, npx will call the audit.js by default instead of calling the command you are passing to the adf-cli.

Reply:
Maybe try the `--` to pass params to the command? like `npm start -- --open` ?

Reply:
It doesn't work.
you can try to run 
`npx @alfresco/adf-cli@alpha update-version --pathPackage "$(pwd)" --version "3.5.0" `

It might be that is because of what we have in the package.json:
```
"bin": {
    "adf-license": "./bin/doc/licenseList.js",
    "adf-audit": "./bin/doc/audit.js",
    "adf-cli": "./bin/adf-cli"
  },
```